content/posts/deploy_private_images.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Deploy private images on Aptible!"
+excerpt: "You can now deploy directly from a private image."
+author_name: Blake Pettersson
+author_email: blake@aptible.com
+author_id: blake
+posted: 2016-07-14
+section: Blog
+posts: true
+---
+We're happy to announce that you can now deploy private images hosted on Docker Hub and quay.io (and other private repositories) on Aptible! This feature is not currently available on our legacy infrastructure. 
+
+## How does it work?
+
+You create an app as usual on Aptible. You will still need to use git, since we still need to have a Procfile. There is no need to have a Dockerfile, as long as you don't need to extend a private Docker image with custom logic. All you have to do is to set a couple of environment variables so we know where to pull the Docker image from.
```

Comment:
I might specify _why_ we need the Procfile. Perhaps something like:

> You will still need to use git, since we will still push a repo with a single Procfile file, used to define the app's services and their commands.